SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -2151,212 +2151,146 @@ void CNEMOEulerSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_container
-//      conv_numerics->SetCvve(nodes->GetCvve(iPoint), node_infty->GetCvve(0));
-//
-//      if (dynamic_grid)
-//        conv_numerics->SetGridVel(geometry->nodes->GetGridVel(iPoint),
-//                                  geometry->nodes->GetGridVel(iPoint));
-//
-//      /*--- Compute the residual using an upwind scheme ---*/
-//      auto residual = conv_numerics->ComputeResidual(config);
-//      LinSysRes.AddBlock(iPoint, residual);
-//
-//      /*--- Jacobian contribution for implicit integration ---*/
-//      //if (implicit)
-//      //  Jacobian.AddBlock2Diag(iPoint, residual.jacobian_i);
-//
-//      /*--- Viscous contribution ---*/
-//      if (viscous) {
-//
-//        /*--- Set the normal vector and the coordinates ---*/
-//        visc_numerics->SetNormal(Normal);
-//        su2double Coord_Reflected[MAXNDIM];
-//        GeometryToolbox::PointPointReflect(nDim, geometry->nodes->GetCoord(Point_Normal),
-//                                           geometry->nodes->GetCoord(iPoint), Coord_Reflected);
-//        visc_numerics->SetCoord(geometry->nodes->GetCoord(iPoint), Coord_Reflected);
-//
-//        /*--- Primitive variables, and gradient ---*/
-//        visc_numerics->SetPrimitive(V_domain, V_inlet);
-//        visc_numerics->SetPrimVarGradient(nodes->GetGradient_Primitive(iPoint), nodes->GetGradient_Primitive(iPoint));
-//
-//        /*--- Laminar viscosity ---*/
-//        visc_numerics->SetLaminarViscosity(nodes->GetLaminarViscosity(iPoint), nodes->GetLaminarViscosity(iPoint));
-//
-//        /*--- Compute and update residual ---*/
-//        auto residual = visc_numerics->ComputeResidual(config);
-//        LinSysRes.SubtractBlock(iPoint, residual);
-//
-//        /*--- Jacobian contribution for implicit integration ---*/
-//        //if (implicit)
-//        //  Jacobian.SubtractBlock2Diag(iPoint, residual.Jacobian_i);
-//      }
-//
-//    }
-//  }
-//
-//  /*--- Free locally allocated memory ---*/
-//  delete [] U_domain;
-//  delete [] U_inlet;
-//  delete [] V_domain;
-//  delete [] V_inlet;
-//  delete [] Normal;
+ unsigned short iDim;
+ unsigned long iVertex, iPoint, Point_Normal;
+
+ bool implicit = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
+ bool dynamic_grid  = config->GetGrid_Movement();
+ bool viscous       = config->GetViscous();
+ string Marker_Tag = config->GetMarker_All_TagBound(val_marker);
+
+ su2double *Normal = new su2double[nDim];
+
+ /*--- Supersonic inlet flow: there are no outgoing characteristics,
+  so all flow variables can be imposed at the inlet.
+  First, retrieve the specified values for the primitive variables. ---*/
+
+ const su2double* Mass_Frac       = config->GetInlet_MassFrac(); 
+ const su2double Temperature      = config->GetInlet_Temperature(Marker_Tag);
+ const su2double Pressure         = config->GetInlet_Pressure(Marker_Tag);
+ const su2double* Velocity        = config->GetInlet_Velocity(Marker_Tag);
+ su2double Temperature_ve   = config->GetInlet_Temperature_ve();
```

Comment:
No pretty formatting please.
```suggestion
 const su2double* Mass_Frac = config->GetInlet_MassFrac(); 
 const su2double Temperature = config->GetInlet_Temperature(Marker_Tag);
 const su2double Pressure = config->GetInlet_Pressure(Marker_Tag);
 const su2double* Velocity = config->GetInlet_Velocity(Marker_Tag);
 const su2double Temperature_ve   = config->GetInlet_Temperature_ve();
```